helium_miner_dashboard.json

```diff
@@ -1,5 +1,14 @@
 {
-  "__inputs": [],
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
```

Comment:
I didn't set this explicitly, but every time I import the original file and then share->export mine, this happens.

Reply:
did you hit the 'externally' option when exporting? I think some of this is just from trying to get it to work for gradoj. Given it imports properly on mine, it must be right.